app/models/language.rb

```diff
@@ -1,4 +1,36 @@
 class Language < ActiveRecord::Base
   has_many :percent_languages
   has_many :percentages, :through => :percent_languages
+
+  def Language.getScopeOf(languages)
```

Comment:
method names should be camel cased. Class methods should either be in `self.class_method` or within a `class << self do ... end` block. 

This is not the standard way of writing them.